### ISSUE TITLE
Add more levels to the game

### ISSUE THREAD

**Issue by gudarzi** [gudarzi/QtSnake#10]
Add more levels to make the game less boring!

**Comment by hmloftus** [comment]
Hello - is this issue still open? I can take a look.

**Comment by gudarzi** [comment]
@hmloftus Yes, of course. Take a look and see if you can add levels to the game.

**Comment by Tselepie** [comment]
Hello, I played a bit with the game and the code. Can I add the fixes and the one extra level I created for you to review and possibly add it to the project?

**Comment by gudarzi** [comment]
@Tselepie Yes, of course. Any contribution to the project is welcome!

**Comment by wzzxd** [comment]
Hi there, I'm interested in working on this. Can I make contributions to this project? Also, are there any other bugs in this that may require fixing?

**Comment by yuval728** [comment]
Hey, I'm new to open source and would like to contribute.

**Comment by yuval728** [comment]
@gudarzi I have created a pull request for adding more levels please review it thankyou.

**Comment by wcxdxd** [comment]
@gudarzi Hi I made a pull request adding new mechanics to the game for added challenge and more fun. Please review and let me know what you think. Thank you!
